### ISSUE TITLE
can u do for the lists feature?

### ISSUE THREAD

**Issue by ucalyptus2** [JeffCarpenter/export-stars#2]
Like export based on listname=machinelearning2022 as an attribute ?

cc: @JeffCarpenter

**Comment by andypiper** [comment]
I don't believe that this is possible in the GitHub API right now (I've been looking for this myself). Hope they make that available in the future!
